starter-code/scripts/article.js

```diff
@@ -2,9 +2,16 @@
 
 var articles = [];
 
-function Article (rawDataObj) {
+function Article (rawData) {
```

Comment:
Doing this can cause issues within the code base, confusion between if it should be looking at your variable or what is here. We do not want to reuse variables. We should also not be editing starter code, because that can break things elsewhere in the code.